packages/client/mutations/toasts/popNotificationToast.ts

```diff
@@ -0,0 +1,80 @@
+import graphql from 'babel-plugin-relay/macro'
+import {Snack} from '../../components/Snackbar'
+import {ValueOf} from '../../types/generics'
+import {OnNextHandler, OnNextHistoryContext} from '../../types/relayMutations'
+import {
+  NotificationEnum,
+  popNotificationToast_notification
+} from '../../__generated__/popNotificationToast_notification.graphql'
+import SetNotificationStatusMutation from '../SetNotificationStatusMutation'
+import mapResponseMentionedToToast from './mapResponseMentionedToToast'
+import mapResponseRepliedToToast from './mapResponseRepliedToToast'
+
+const typePicker = {
+  KICKED_OUT: null,
+  PAYMENT_REJECTED: null,
+  TASK_INVOLVES: null,
+  PROMOTE_TO_BILLING_LEADER: null,
+  TEAMS_LIMIT_EXCEEDED: null,
+  TEAM_ARCHIVED: null,
+  TEAM_INVITATION: null,
+  MEETING_STAGE_TIME_LIMIT_END: null,
```

Comment:
@jmtaber129 I assume we didn't have all these notifications below, and it's just a placeholder to introduce them in the future?

Reply:
This was initially sort of some copy-pasta from `NotificationPicker`, but I realized it's better to just declare the type as a `Partial<Record<>>` instead of explicitly marking these as `null`.

Went ahead and updated `NotificationPicker` to reflect this, too.